lib/sqs.js

```diff
@@ -44,6 +76,49 @@ module.exports = ({ account, queuePrefix = '', queueSuffix = '', defaultVisibili
      */
     remove: ({ queueName, entries }) => SQS.deleteMessageBatchAsync({ Entries: _.uniqBy(entries, 'Id'), QueueUrl: getQueueURL(queueName) }),
 
+    /**
+     * Retrieves an extended message to a queue, meaning: if the message is
```

Comment:
this will actually return an array of up to 10 extended messages right?

Reply:
👍 or if a different `max` is specified